src/images/eventstoredb.rs

```diff
@@ -0,0 +1,102 @@
+use crate::{core::WaitFor, Image};
+use std::collections::HashMap;
+
+const CONTAINER_IDENTIFIER: &str = "eventstore/eventstore";
+const DEFAULT_TAG: &str = "20.10.0-bionic";
+
+#[derive(Debug, Default, Clone)]
+pub struct EventStoreDBArgs;
+
+impl IntoIterator for EventStoreDBArgs {
+    type Item = String;
+    type IntoIter = ::std::vec::IntoIter<String>;
+
+    fn into_iter(self) -> <Self as IntoIterator>::IntoIter {
+        vec![].into_iter()
+    }
+}
+
+#[derive(Debug)]
+pub struct EventStoreDB {
+    tag: String,
+    arguments: EventStoreDBArgs,
+    env_vars: HashMap<String, String>,
+}
+
+impl EventStoreDB {
+    pub fn insecure_mode(mut self) -> Self {
+        self.env_vars
+            .insert("EVENTSTORE_INSECURE".to_string(), "true".to_string());
+        self.env_vars.insert(
+            "EVENTSTORE_ENABLE_ATOM_PUB_OVER_HTTP".to_string(),
+            "true".to_string(),
```

Comment:
I guess these could actually use `self.add_env_var` to avoid some of the duplication?